production-config/README.md

```diff
@@ -0,0 +1,59 @@
+## Deploy OpenFisca-Web-API in production
+
+> This how-to explains how to host OpenFisca-Web-API in *production*, for example to serve `api.xxx.com`.
+> If you just need to develop with a local install of OpenFisca-Web-API, you need to read
+> the main [README](https://github.com/openfisca/openfisca-web-api/blob/master/README.md).
+
+The official and public instance of the API available at `https://api.openfisca.fr/` is hosted using:
+- [Debian Jessie](https://www.debian.org/News/2015/20150426) GNU/Linux distribution
+- [Nginx](https://www.nginx.com/) web server
+- [gunicorn](http://gunicorn.org) WSGI-compatible Python web server
```

Comment:
This paragraph should probably be moved down, if I want to run my own instance I don't care that much about what the official is using.

If, on the other hand, you mean that the documentation is for this setup only, then please make it more explicit 🙂